package.json

```diff
@@ -6,6 +6,8 @@
   "description": "An automation task to compile css-modules and remove it from the components",
   "scripts": {
     "test": "NODE_ENV=test mocha --compilers js:babel-register --reporter emoji-reporter ./test/**/*.spec.js",
+    "coverage": "NODE_ENV=test nyc --cache --require babel-core/register mocha ./test/**/*.spec.js --timeout=8000",
```

Comment:
You're right, I think the problem is in how I'm invoking mocha trough `nyc`. I've updated the scripts [here](https://github.com/cef62/css-modules-compiler/pull/23/commits/6261b166b4ff657d963936e6635099871e5950e0) and it works perfectly now!